### PR TITLE
Allows get the country's code giving its name

### PR DESCRIPTION
**Pull request by dgdavid**

### CODE REVIEW

File: src/iso-3166-country-codes-angular.js [hunk 1]
```diff
@@ -263,6 +269,13 @@ angular.module('iso-3166-country-codes', [])
       return angular.isDefined(this.codeToCountry[input]);
     };
 
+    holder.getCountryCode = function(countryName, manipulator) {
+      var countryCode = this.countryToCode[countryName.toUpperCase()];
+      manipulator = manipulator ? manipulator : 'toLowerCase';
```
Comment (rsertelon) [inline src/iso-3166-country-codes-angular.js]: Why did you choose `toLowerCase` as default manipulator?

ISO codes are uppercase by default :)

**Comment by rsertelon** [comment]
Hi,

Thanks for you PR :)

I've just one question for you, it's in the line notes ;)

Thanks!

File: src/iso-3166-country-codes-angular.js (hunk 1 above)
Comment (dgdavid) [inline src/iso-3166-country-codes-angular.js]: Ups!

Excuse me, I was wrong. I thought that the code should be returned in lowercase, but  actually it's in uppercase according to ISO-3166. Could you fix it?

Thanks!.

Comment (rsertelon) [inline src/iso-3166-country-codes-angular.js]: No problem, will do and merge. Thanks for your contribution!

**Comment by rsertelon** [comment]
Hmm I've just seen that this pull request is targeted on the `master` branch. In this project, I use the `develop` branch as default branch.

I won't merge this PR as is, but will cherry-pick you fix onto `develop`.

Thanks again ;)

**Comment by rsertelon** [comment]
I've cherry-picked your commit, but I forgot to ask you if you can confirm that this statement is okay for you:

```
I dedicate any and all copyright interest in this software to the
public domain. I make this dedication for the benefit of the public at
large and to the detriment of my heirs and successors. I intend this
dedication to be an overt act of relinquishment in perpetuity of all
present and future rights to this software under copyright law.
```

If so, please add it to the description of your PR.

Thanks !

**Comment by dgdavid** [comment]
Ok @rsertelon, done :)

Thanks!

**Comment by rsertelon** [comment]
Thank you ;) BTW, I released 1.1.1 with your contribution :)

**Pull request closed without merging**